app/src/androidTest/java/ch/epfl/balelecbud/FestivalInformationFragmentTest.java

```diff
@@ -53,13 +54,31 @@ public void testCanAddInfoToDatabase() {
         testInfoInView(onView(new RecyclerViewMatcher(R.id.festivalInfoRecyclerView).atPosition(0)), info);
     }
 
+    @Ignore
```

Comment:
Why add a test and ignore it ? Besides it should be called something else like "deletedInfoIsNotDisplayed" the title implies we're testing the mock DB